alacritty.yml

```diff
@@ -205,6 +205,9 @@ key_bindings:
 mouse_bindings:
   - { mouse: Middle, action: PasteSelection }
 
+multiclick_threshold_ms: 300
```

Comment:
I think `multiclick_threshold_ms` should probably put under `mouse` config if anything.

I can put the `escape_chars` under `selection`, but it wouldn't make much sense right now since it's the only field, (unless we want to make selection more customizable).